src/css/base.styl

```diff
@@ -297,6 +289,14 @@ button.off
 button .fake-link
     color: white-solid
 
+.casual
```

Comment:
Any reason to move these away from the other legality-related colors?

Reply:
Yeah, its due to the way CSS chooses what property to use when classes are equal. For background, for all of our card lines in the deck builder we use the `.fake-link` tag to color them orange and make the mouse cursor turn into a pointer when you hover over it. When a card in the deck builder is rotated or banned we change the span's class to `.inactive` or `.casual` to change the color, but in the process would lose the pointer on hover. I wanted to keep the pointer aspect of `.fake-link` while still changing the color, but due to the CSS priority of the two classes being the same it would prioritize the color of `.fake-link` because it was defined later in the CSS file. That was a long way of saying that I moved those below `.fake-link` so I can use the colors from it.

This was not the cleanest change. I should revisit this to at least organize it better. Maybe move these to their own file or move all the legality colors to the same spot. Maybe even better, just remove the color aspect from `.fake-link` and default the color to the orange elsewhere.

Reply:
Hmm yeah, that's definitely awkward. css sucks, lol. I'd say move all of the legality colors in that case, better to keep everything grouped.